src/wp-includes/class-wp-block-type.php

```diff
@@ -422,6 +430,11 @@ public function __set( $name, $value ) {
 			return;
 		}
 
+		if ( 'uses_context' === $name ) {
```

Comment:
Now that I look more closely, I believe we can get rid of both `variations` and `uses_context` conditions here. It will be handled by the next block for non-deprecated properties.

Reply:
I removed it in [this commit](https://github.com/WordPress/wordpress-develop/pull/6079/commits/0c4a795247b372b5efb103350ae532dc9058d584). Let me know if that's not what you suggested.